.github/workflows/building-bundle.yml

```diff
@@ -0,0 +1,40 @@
+name: Building bundle
+
+on: [ push, pull_request ]
+
+jobs:
+  build-all:
+    name: 'Building'
+
+    runs-on: ${{ matrix.os }}
+
+    strategy:
+      matrix:
+        os: [ windows-latest, ubuntu-latest, macos-latest ]
```

Comment:
No need for matrix builds, no native code is used in extension. Just keep `windows-latest` @acidicMercury8

Reply:
@PeyTy okay, fixed